assets/js/atomic/blocks/product/block-components.js

```diff
@@ -0,0 +1,7 @@
+export { default as ProductButton } from './button/block';
+export { default as ProductImage } from './image/block';
+export { default as ProductPrice } from './price/block';
+export { default as ProductRating } from './rating/block';
+export { default as ProductSaleBadge } from './sale-badge/block';
+export { default as ProductSummary } from './summary/block';
+export { default as ProductTitle } from './title/block';
```

Comment:
This file is used in the block->component map. We map block name to the block component itself.